### ISSUE TITLE
Inventory sprites not rendering (Legend of Zelda: Ocarina of Time 3D)

### ISSUE THREAD

**Issue by Jamish** [citra-emu/citra#3771]
Nightly Build: 45aa950
![image](https://user-images.githubusercontent.com/3383460/40408796-923cfb34-5e1d-11e8-8ed0-ada9b2342d5b.png)

As you can tell, on the bottom (right) screen, none of the item sprites are rendering. I can confirm this was working when I first downloaded and installed the emulator on May 13th (I am not sure which build that corresponds to exactly)

Log dump:
```
[   9.933880] Frontend <Info> citra_qt/main.cpp:BootGame:647: Citra starting...
[  10.190747] Input <Error> input_common/sdl/sdl.cpp:SDLJoystick:35: failed to open joystick 0
[  10.191917] Render.OpenGL <Info> video_core/renderer_opengl/renderer_opengl.cpp:Init:488: GL_VERSION: 3.3.0 NVIDIA 391.35
[  10.191921] Render.OpenGL <Info> video_core/renderer_opengl/renderer_opengl.cpp:Init:489: GL_VENDOR: NVIDIA Corporation
[  10.191922] Render.OpenGL <Info> video_core/renderer_opengl/renderer_opengl.cpp:Init:490: GL_RENDERER: GeForce GTX 1080/PCIe/SSE2
[  10.225349] Loader <Info> core/loader/ncch.cpp:Load:159: Program ID: 0004000000033500
[  10.225457] Service.FS <Warning> core/file_sys/ncch_container.cpp:OpenFile:113: Failed to open C:\Users\Jamison\AppData\Roaming/Citra/sdmc/Nintendo 3DS/00000000000000000000000000000000/00000000000000000000000000000000/title/0004000e/00033500/content/00000000.app
[  10.259318] Service.FS <Info> core/file_sys/ncch_container.cpp:LoadSectionExeFS:275: Attempting to load logo from the ExeFS
[  10.259560] Service.CFG <Info> core/hle/service/cfg/cfg.cpp:SetPreferredRegionCode:602: Preferred region code set to 1
[  10.337095] Service.SRV <Warning> core/hle/service/sm/srv.cpp:RegisterClient:48: (STUBBED) called
[  10.337578] Service.SRV <Warning> core/hle/service/sm/srv.cpp:EnableNotification:70: (STUBBED) called
[  10.340071] Service.APT <Warning> core/hle/service/apt/apt.cpp:GetLockHandle:255: (STUBBED) called applet_attributes=0X00000000
[  10.368044] Service.APT <Warning> core/hle/service/apt/apt.cpp:NotifyToWait:234: (STUBBED) app_id=768
[  10.371504] Service.APT <Warning> core/hle/service/apt/apt.cpp:AppletUtility:455: (STUBBED) called command=0X00000007, input_size=0X00000004, output_size=0X00000001
[  10.372269] Service.APT <Warning> core/hle/service/apt/apt.cpp:AppletUtility:455: (STUBBED) called command=0X00000004, input_size=0X00000001, output_size=0X00000001
[  10.400103] Service.GSP <Warning> core/hle/service/gsp/gsp_gpu.cpp:AcquireRight:665: called flag=00000000 process=10 thread_id=0
[  10.451802] Service.DSP <Info> core/hle/service/dsp_dsp.cpp:LoadComponent:150: Firmware hash: 0x4b1ec1487e523c5a
[  10.451806] Service.DSP <Info> core/hle/service/dsp_dsp.cpp:LoadComponent:154: Structures hash: 0xd194fc96db654a27
[  10.451808] Service.DSP <Warning> core/hle/service/dsp_dsp.cpp:LoadComponent:159: (STUBBED) called size=0xBE0C, prog_mask=0x000000FF, data_mask=0x000800FF, buffer=0x0054CC8C
[  10.452505] Service.DSP <Info> core/hle/service/dsp_dsp.cpp:RegisterInterruptEvents:251: Registered type=2, pipe=2, event_handle=0x0009001E
[  10.453239] Service.DSP <Warning> core/hle/service/dsp_dsp.cpp:GetSemaphoreEventHandle:176: (STUBBED) called
[  10.453516] Service.DSP <Warning> core/hle/service/dsp_dsp.cpp:SetSemaphoreMask:473: (STUBBED) called mask=0x00002000
[  10.454035] Audio.DSP <Info> audio_core/hle/hle.cpp:PipeWrite:149: Application has requested initialization of DSP hardware
[  10.454197] Service.DSP <Warning> core/hle/service/dsp_dsp.cpp:SetSemaphore:274: (STUBBED) called
[  10.457576] Service.DSP <Warning> core/hle/service/dsp_dsp.cpp:SetSemaphore:274: (STUBBED) called
[  10.487279] Kernel.SVC <Error> core/hle/kernel/svc.cpp:CreateThread:750: Newly created thread must run in the SysCore (Core1), unimplemented.
[  10.525662] Service <Error> core/hle/service/service.cpp:ReportUnimplementedFunction:163: unknown / unimplemented function 'SetInternalPriorities': port='gsp::Gpu' cmd_buf={[0]=0x1e0080, [1]=0x19, [2]=0x1a}
[  11.210798] Service.HID <Warning> core/hle/service/hid/hid.cpp:GetGyroscopeLowCalibrateParam:342: (STUBBED) called
[  11.387471] Service.NDM <Warning> core/hle/service/ndm/ndm.cpp:SuspendScheduler:115: (STUBBED) called
[  11.387634] Service.APT <Warning> core/hle/service/apt/apt.cpp:AppletUtility:455: (STUBBED) called command=0X00000004, input_size=0X00000001, output_size=0X00000001
[  16.416668] Service.FS <Error> core/file_sys/savedata_archive.cpp:OpenFile:67: Non-existing file C:\Users\Jamison\AppData\Roaming/Citra/sdmc/Nintendo 3DS/00000000000000000000000000000000/00000000000000000000000000000000/title/00040000/00033500/data/00000001/save01.bin can't be open without mode create.
[  16.416675] Service.FS <Error> core/hle/service/fs/fs_user.cpp:OpenFile:69: failed to get a handle for file [Wchar: /save01.bin]
[  16.417099] Service.FS <Error> core/file_sys/savedata_archive.cpp:OpenFile:67: Non-existing file C:\Users\Jamison\AppData\Roaming/Citra/sdmc/Nintendo 3DS/00000000000000000000000000000000/00000000000000000000000000000000/title/00040000/00033500/data/00000001/save02.bin can't be open without mode create.
[  16.417944] Service.FS <Error> core/hle/service/fs/fs_user.cpp:OpenFile:69: failed to get a handle for file [Wchar: /save02.bin]
[  30.954347] Service.CAM <Info> citra_qt/camera/qt_multimedia_camera.cpp:StopCameras:199: Stopping all cameras
```

Running Windows 10 (Version 1803)
Nvidia driver version 391.35

Let me know if there's anything else needed!

**Comment by Hexagon12** [comment]
Enable Accurate Multiplication in Emulation - Configure - Graphics for a workaround.

**Comment by Jamish** [comment]
Thanks, that worked. Apologies if this is a duplicate issue; I did a quick search of open issues and couldn't find much related.

**Comment by FearlessTobi** [comment]
Is this really an issue if it can be fixed by accurate multiplication, or can this be closed?

**Comment by Jamish** [comment]
I don't remember changing that checkbox when I first installed Citra on May 13th, and everything rendered properly.

I just went back and tried out the Nightly Build from back then. Turns out the Graphics menu looks different, and `Enable Accurate Multiplication` wasn't even an option. Looks like it was always enabled, and now it's got an option that's disabled by default. 

No wonder I don't remember changing the checkbox :) 

Hope you don't mind me closing this issue. Though, as as suggestion, I think `Enable Accurate Multiplication` should default to being on, at least for the games that require it (such as OOT:3D).
